helpdesk/models/helpdesk_ticket.py

```diff
@@ -83,6 +88,20 @@ def _onchange_partner_id(self):
             self.partner_name = self.partner_id.name
             self.partner_email = self.partner_id.email
 
+    @api.multi
+    @api.onchange('team_id', 'user_id')
+    def _onchange_dominion_user_id(self):
```

Comment:
I still don't get how this works. Am I missing something? Where's this domain applied?